post-processing/fuzz_analysis.py

```diff
@@ -234,6 +252,10 @@ def get_hit_count_color(hit_count):
         n1.cov_forward_reds = forward_red
         n1.cov_largest_blocked_func = largest_blocked_name
 
+    branch_blockers = detect_branch_level_blockers(profile, branch_profiles)
+    logger.info(f"[+] found {len(branch_blockers)} branch blockers.")
+    # TODO: use these results appropriately ...
```

Comment:
Am curious, is the goal to use this based on the logging for now? I'm asking because I didn't see the explicit use of the stuff for now